src/components/DrivePicker.tsx

```diff
@@ -82,6 +83,9 @@ const DrivePicker = () => {
         )}
       </GooglePicker>
       {renderEmbed()}
+
+      { spreadsheet && spreadsheet.sheets ? <DocTabsRadioSection spreadsheet={spreadsheet}/> : null }
```

Comment:
🤷‍♂ This can be simplified

```suggestion
      { (spreadsheet && spreadsheet.sheets) && <DocTabsRadioSection spreadsheet={spreadsheet}/> }
```